### DISCUSSION TITLE
maximized window revert to normal when i click other window

### DISCUSSION THREAD

**Discussion by js210711** [Xpra-org/xpra#4095] [Q&A]
when i click other window of x app via xpra or windows app, maximized window revert to normal. any way to prevent this behavior of reverting to normal size? If no configuration is available now, I wish this feature is implemented. Thanks in advance for help.

**Comment by js210711** [comment]
while checking this problem further, I have got to know further on this problem.
- occurs in xpra v6.0-r34898 (gda267883c) beta.
- is not shown in xpra v5.0.4-r0

**Reply by totaam** [reply]
Is this an issue with the 6.0 client or the 6.0 server?

**Reply by js210711** [reply]
the symtom is in client side. X apps delivered via xpra revert to normal size when I click other app, X app or Windows app.

I am using xpra client in windows PC. Windows Xpra version is 5.0.4. so xpra connecting to remote server running xpra 6.0 beta have this problem..

If Xpra client 6.0 beta built for Windows is available, it may be worthwhile to try.

**Reply by js210711** [reply]
I have just installed Xpra 6.0 for windows, but the problem of reverting to normal size when I click other app is the same as before. any configuration tweak will be helpful to resolve this annoying problem.

**Reply by totaam** [reply]
Please provide a sample open source application that I can use to reproduce the problem.

**Comment by js210711** [comment]
I am not sure what you mean on sample open source application.

Here is my use configuration.


local machine: Xpra client at Windows 11 PC, two display attached to local machine with extended display mode, xpra 5.0, 6,0 both tested
remote machine: rocky linux 8.9, xpra 6.0 beta

any X app delivered from remote machine via xpra has this problem. I normally maximize all the app for use. and I am switching app by clicking other app in other display or windows taskbar.

**Comment by totaam** [comment]
Sample: can you reproduce with an `xterm`?
Can you reproduce with a single monitor?

**Comment by totaam** [comment]
Not heard back, closing.
